README.md

```diff
@@ -151,6 +164,28 @@ If everything is running correctly, you should get a response with information a
 }
 ```
 
+## Playground
+
+OpenFGA includes a playground use to facilitate the creation and visualisation of authorization models and tuples within your running local instance.
+
+Once OpenFGA is running, the playground can be accessed at [http://localhost:3000/playground].
+
+In the event that a port other than the default `:3000` is required, the `OPENFGA_PLAYGROUND_PORT` environment variable can be set when starting the instance. E.g.,
+
+**Pre-compiled Binaries**
+
+```sh
+OPENFGA_PLAYGROUND_PORT=3001 ./bin/openfga
+```
+
+**Source**
+
+```sh
+OPENFGA_PLAYGROUND_PORT=3001 ./openfga
+```
+
```

Comment:
Should we add a docker section with:

```sh
docker run -p 8080:8080 -p 3000:3000 openfga/openfga run
```

Although, I'll mention it again, I think we should simplify our README and the many ways of running and building that we are documenting. I think we should make use of our Makefile and let that be the documentation. What does everyone think?

Reply:
Personally, I don't love make target bloat such as `make run`, `make run-playground`, `make run-postgres`, `make run-otherA`, etc.. Config should be easy to change and manage, and if it isn't then we should be prioritizing that problem.

I think the Makefile's "North Star" should always be to assist with common, and tedious, developer setup and development tasks. It shouldn't just become a dumping ground for alternate make targets that just set a config value differently before invoking the binary. 

This is obviously my opinion, and I think it's worth talking about more generally, but I think we should strive to discourage make targets that are just aliases for running the server with a particular config value. I think it's ok to ask a developer to change config if they want to run the server with different config.

Reply:
Actually, I agree with you. What I was thinking more of is that we know have quite a number of ways to run OpenFGA in the readme and, in my opinion, it would be ideal to have (the Docker way of course and then) one easy way such as `make run`, and that would build and execute. If a user needs more details they can take a look at the Makefile. So the Makefile documents the steps to build from source, and we can remove that from the README, where I think it is not necessary.

Once we have an easy to set config (as in your PR) then it makes sense to to trim what we have now in the Makefile. As it is currently, since we don't read from a `.env` file, and finding out what env variables we can set, I think having a bunch of Make targets is not actually that bad a compromise.

Reply:
> As it is currently, since we don't read from a .env file, and finding out what env variables we can set, I think having a bunch of Make targets is not actually that bad a compromise.

I agree. I think once we get these config changes we can start moving away from this pattern like you said as well.

> Actually, I agree with you. What I was thinking more of is that we know have quite a number of ways to run OpenFGA in the readme and, in my opinion, it would be ideal to have (the Docker way of course and then) one easy way such as make run, and that would build and execute.

I get what you're saying, and I like the intent behind it a lot. I think the challenges comes from the fact that we have different installation/setup methods, and having a docker target and a `make run` target leaves out the folks that may want to run the pre-compiled binary. I think for the source code installation instructions we could simply focus on the `make run` approach. But we do have

* Docker
* Pre-compiled Binaries
* Building from Source

I think we could probably drop the `docker-compose` instructions and just prioritize the docker instructions, but drop a hint that you can use the docker-compose file included in the repository if you like to use docker-compose 🤷 ?